hydra-node/test/Hydra/HeadLogicSnapshotSpec.hs

```diff
@@ -199,13 +200,14 @@ prop_singleMemberHeadAlwaysSnapshotOnReqTx sn = monadicST $ do
         , localTxs = []
         , confirmedSnapshot = sn
         , seenSnapshot
+        , decommitTx = Nothing
         }
     outcome = update aliceEnv simpleLedger (inOpenState' [alice] st) $ NetworkEvent defaultTTL alice $ ReqTx tx
     Snapshot{number = confirmedSn} = getSnapshot sn
     nextSn = confirmedSn + 1
   pure $
     ( collectEffects outcome
-        `shouldContain` [NetworkEffect (ReqSn nextSn [txId tx])]
+        `shouldContain` [NetworkEffect (ReqSn nextSn [txId tx] Nothing)]
```

Comment:
i see all tests in this module having Nothing as value for `decommitTx`.

should not we have a test where it is set to Just?

Reply:
We just populated the new field to please the compiler. We will add tests for `ReqSn` with some decommit too.

Reply:
You can take a look at the `HeadLogicSpec`, there we added some test using `ReqSn` with populated `decommitTx`.